src/tools/illink/PATENTS.TXT

```diff
@@ -0,0 +1,44 @@
+Microsoft Patent Promise for Mono
```

Comment:
Is this needed?  (The root of the repo has a different PATENTS.TXT.)

cc: @richlander